backend/core/src/listings/entities/listing.entity.ts

```diff
@@ -136,6 +136,12 @@ class Listing extends BaseEntity {
   @IsBoolean({ groups: [ValidationsGroupsEnum.default] })
   referralOpportunity?: boolean
 
+  @Column({ type: "boolean", nullable: true })
+  @Expose()
+  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
+  @IsBoolean({ groups: [ValidationsGroupsEnum.default] })
+  section8Acceptance?: boolean | null
```

Comment:
I think it's OK in this case to not default to false since we will only query where section 8 is true. If we were needing to query on a not true state, then we would want to default this to false.